packages/nimble-components/src/rich-text-viewer/tests/rich-text-viewer-matrix.stories.ts

```diff
@@ -109,6 +109,10 @@ export const differentContentsInMobileWidth: StoryFn = createStory(html`
             [
                 'Long word',
                 'ThisIsALongWordWithoutSpaceToTestLongWordInSmallWidth'
+            ],
```

Comment:
nit: Out of scope Typo. Change `sup` to `sub` for all instances in this or other files

![image](https://github.com/ni/nimble/assets/94439533/c3f1314b-3c02-4fb2-8b1d-ec81d5fc9d52)